shared/lib-bie-kafka/build.gradle

```diff
@@ -0,0 +1,12 @@
+plugins {
+  id 'local.std.java.library-conventions'
+}
+
+ext {
+  // TODO: improve test code coverage so that the following can be removed
```

Comment:
This would be great on a motivational poster in an office.  Or as a title of story.  But in source, I don't think it makes it any easier for the reader of the code.